fred2/orienteditor.cpp

```diff
@@ -21,7 +21,8 @@
 static char THIS_FILE[] = __FILE__;
 #endif
 
-const float input_threshold = 0.01f;		// smallest increment of input box
+constexpr auto INPUT_THRESHOLD = 0.01f;		// smallest increment of input box
+constexpr auto INPUT_FORMAT = "%.01f";
```

Comment:
Based on [this page](https://medium.com/@samathy_barratt/constantly-confusing-c-const-and-constexpr-pointer-behaviour-7eddb041e15a), `constexpr char *` is equivalent to `char * const` and not `const char *`.

I think either `const auto` or `const auto const` would be better.

Reply:
That page covers `constexpr char *`, but not `constexpr auto`.  String literals, written out in the code, always have the type of `const char *`.  So I'm pretty sure that `auto` would interpret them correctly.

Reply:
Looks like Visual Studio's tool tip on similar code agrees with you. So `constexpr auto` it is.